### PR TITLE
Add profile age and user settings now work

### PR DESCRIPTION
**Pull request by TruddyTheDuddi**
**What**
User profile settings with checks, by class `UserSettingsViewModel` and `UserSettingsActivity`, updates successfully in the database. For now it uses a predefined UUID, and there's no image uploading.

Added birthday field to user profile (an optional field) and can be null in database. 

Added a function in the class `Time` to open an only-date selector.

**New**
Added a function in `ProfileService` and the associated classes to check if a user exists.

**NOTE** I pulled Louca's `ui.login` to get the variable names for the intent and fixed a small bug, so it added it as modified, but you can skip it

### CODE REVIEW

**Comment by TruddyTheDuddi** [comment]
No tests for now, but these will be the next step

**Comment by TruddyTheDuddi** [comment]
I am opening the reviews, if it didn't happen by the time you review, I will deal with the magic strings properly some other time

File: app/src/androidTest/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetupViewTest.kt [hunk 1]
```diff
@@ -228,73 +228,73 @@ class MeetupViewTest {
...
-                RecyclerViewMatcher(R.id.profile_list_recycler).atPositionOnView(
-                    0,
-                    R.id.profile_name
-                )
-            )
-                .perform(click())
-        }
-
-    }
+//    @Test
```
Comment (atappy) [inline app/src/androidTest/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetupViewTest.kt:231]: this first test should be working, it was only the second that kept crashing for me

File: app/src/main/java/com/github/palFinderTeam/palfinder/user/settings/UserSettingsActivity.kt [hunk 2]
```diff
@@ -1,16 +1,247 @@
...
+     */
+    private fun bindFieldsToData() {
+        forwardBind()
+        backwardsBind()
+    }
+
+    /**
+     * Bind fields to view model
+     */
+    private fun forwardBind() {
```
Comment (atappy) [inline app/src/main/java/com/github/palFinderTeam/palfinder/user/settings/UserSettingsActivity.kt:95]: I don't understand why you need to have a forward and a backward bind

**Review by atappy — APPROVED** [review]
otherwise good job !

**Review by Gerussi — APPROVED** [review]
Looks good to me

**Pull request merged**